python/src/opendp/_convert.py

```diff
@@ -690,35 +729,56 @@ def _check_polars_by(by):
 
 
 def _margin_to_slice(val) -> FfiSlicePtr:
+    from opendp.extras.polars import Margin
+    assert isinstance(val, Margin)
```

Comment:
Confirm that this is an assertion about our logic, and not a validation of anything coming from the user?